packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -339,7 +340,13 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
   async addStatusChangeEvent() {
     const azureAccount: AzureAccount =
       vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account")!.exports;
+    AzureAccountManager.currentStatus = azureAccount.status;
     azureAccount.onStatusChanged(async (event) => {
+      if(AzureAccountManager.currentStatus === "Initializing") {
+        AzureAccountManager.currentStatus = azureAccount.status;
```

Comment:
AzureAccountManager.currentStatus = **event**

The same for line 349

Reply:
Fixed.